src/cc/bcc_syms.cc

```diff
@@ -13,8 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#ifndef _GNU_SOURCE
```

Comment:
don't remember. some glibc function is only available if this is defined.
we can probably drop it here.